crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -26,7 +26,7 @@ def initialize(backup)
       end
 
       def restore
-        [:crowbar, :knife, :database].each do |component|
+        [:restore_crowbar, :run_installer, :restore_chef, :restore_database].each do |component|
```

Comment:
I have taken @vuntz suggestion [here](https://github.com/crowbar/crowbar-core/pull/144/files#r49711356)
to me personally it does not really matter